hardhat-configs/networks.ts

```diff
@@ -1,22 +1,22 @@
-import { INFURA_API_KEY, RPC } from "@kreskolabs/configs";
+import { INFURA_API_KEY, RPC_URL, ETHERSCAN, chains } from "@kreskolabs/configs";
 import { parseUnits } from "ethers/lib/utils";
 import { HttpNetworkConfig, NetworksUserConfig } from "hardhat/types";
 
+/** @deprecated */
```

Comment:
What exactly does this refer to? If something is deprecated it may be good to include a comment regarding what should be used instead.

Reply:
ah right. 

purely use the tag for the visual feedback, lazy

![image](https://user-images.githubusercontent.com/18403470/230967710-60c73188-a347-4553-8c87-1d783b4e204f.png)

but definitely agree a description should be written if it is pushed 👍